internal/errs/silencederror.go

```diff
@@ -0,0 +1,24 @@
+package errs
+
+import (
+	"errors"
+)
+
+type silencedError struct {
+	error
```

Comment:
I know you just copied this, but please make this a named property instead, and satisfy the error interface manually.

Without this Unwrap would effectively return the same error, which could result in funky behaviour.